JornsenChao/kai-squad-nebula-navigators-final
# Pull Request - Epic 13 Task 6 Implement Quiz Creation Error Notifications

## Summary
This pull request implements error notifications during file upload in the quiz creation process. The feature rejects non-PDF files and displays an error notification indicating the number of files that failed to upload, guiding the user to upload PDF files.

## Changes
- Added logic to reject non-PDF files during file upload in quiz creation.
- Integrated error notifications to inform users about invalid file formats and the number of failed uploads.
- Enhanced user experience by guiding users to upload the correct file format.

## Testing
- Verified that non-PDF files are correctly rejected and do not proceed with the upload.
- Confirmed that error notifications display the correct number of failed uploads.
- Ensured the dismiss functionality (close the notification) works as expected.

## Results
The testing results confirm that the error notification system works as intended.

## Notes
- This feature is designed to enhance user feedback during the quiz creation process, reducing the chances of incorrect file submissions.

## Screenshots
![image](https://github.com/user-attachments/assets/127195bf-8a92-4c1e-bcd8-8e4038a90d7e)

## How to Test
1. Start the application and navigate to the quiz creation page.
2. Attempt to upload non-PDF files (e.g., .docx, .jpg).
3. Check the error notification for failed uploads.
4. Close the notification or wait for it to disappear after 6 seconds.
